src/core/src/core_logic/PatchInstaller.py

```diff
@@ -98,6 +106,16 @@ def start_installation(self, simulate=False):
 
         return overall_patch_installation_successful
 
+    def write_installer_perf_logs(self, patch_operation_successful, installed_patch_count, number_of_rounds, maintenance_window, maintenance_window_exceeded, task_status, error_msg):
+        perc_maintenance_window_used = maintenance_window.get_percentage_maintenance_window_used()
+
+        patch_installation_perf_log  = {Constants.LogStrings.TASK: Constants.INSTALLATION, Constants.LogStrings.PACKAGE_MANAGER: self.package_manager_name,
+                                        Constants.LogStrings.PATCH_OPERATION_SUCCESSFUL: str(patch_operation_successful), Constants.LogStrings.INSTALLED_PATCH_COUNT: str(installed_patch_count),
+                                        Constants.LogStrings.NUMBER_OF_ROUNDS: str(number_of_rounds), Constants.LogStrings.MAINTENANCE_WINDOW: str(maintenance_window.duration),
+                                        Constants.LogStrings.PERC_MAINTENANCE_WINDOW_USED: str(perc_maintenance_window_used), Constants.LogStrings.MAINTENANCE_WINDOW_EXCEEDED: str(maintenance_window_exceeded),
+                                        Constants.LogStrings.TASK_STATUS: task_status, Constants.LogStrings.ERROR_MSG: error_msg}
```

Comment:
very minor: add TASK_STATUS right after, TASK. Improved readability and avoids confusions sine we also have PATCH_OPERATION_SUCCESSFUL

Reply:
Done